electrum/gui/qml/components/wizard/WCHaveSeed.qml

```diff
@@ -221,13 +225,14 @@ WizardComponent {
                 id: extendcb
                 Layout.columnSpan: 2
                 Layout.fillWidth: true
+                visible: _seedType && _seedType != '2fa'
```

Comment:
This `seedType != '2fa'` comparison is now duplicated all over the place. I count 4 instances in this PR.
Could this not be moved e.g. to a method of the wizard instead?